src/sass/layout/_header.scss

```
@@ -0,0 +1,156 @@
+// Logo header
+.header-logo img,
+.logo-modal img {
+  @include for-size(desktop) {
+    width: 130px;
+    height: 48px;
+  }
+}
+.page-header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.header-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  // margin-bottom: 54px;
+
+  @include for-size(tablet) {
+    // margin-bottom: 64px;
+  }
+
+  @include for-size(desktop) {
+    // margin-bottom: 46px;
+  }
+}
+.header-nav-contact {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.header-list {
+  display: none;
+
+  @include for-size(desktop) {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    // max-width: 570px;
+    list-style: none;
+    margin-right: 42px;
+    padding: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+}
+.header-item {
+  @include for-size(desktop) {
+    letter-spacing: 0.6px;
+  }
+
+  &:not(:last-child) {
+    margin-right: 40px;
+  }
+}
+.header-link {
+  font-family: 'Montserrat', sans-serif;
+  text-decoration: none;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 1.8;
+  letter-spacing: 0.03em;
+
+  color: $title-white;
+  transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+  @include for-size(tablet) {
+    font-weight: 800;
+    font-size: 26px;
+    line-height: 1.8;
+    letter-spacing: 0.03em;
+  }
+  @include for-size(desktop) {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 1.2;
+    letter-spacing: 0.03em;
+    text-transform: uppercase;
+    color: $text-light;
+  }
+  &:hover,
+  &:focus {
+    color: $accent;
+  }
+}
+// button top header
+.header-btn-consult {
+  display: none;
+
+  &:hover,
+  &:focus {
+    color: $title-white;
+    background-color: $accent;
+    border: 2px solid $accent;
+  }
+  @include for-size(tablet) {
+    display: block;
+
+    min-width: 260px;
+    max-height: 48px;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 1.16;
+    letter-spacing: 0.08em;
+    text-transform: uppercase;
+    text-align: center;
+    padding: 15px 5px;
+
+    color: $accent;
+    border-color: transparent;
+    border: 2px solid $accent;
+    background: 0 0;
+
+    margin-right: 45px;
+
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1),
+      background-color 250ms cubic-bezier(0.4, 0, 0.2, 1),
+      border 250ms cubic-bezier(0.4, 0, 0.2, 1);
```

Потерялось `:` после background color

```suggestion
      border: 250ms cubic-bezier(0.4, 0, 0.2, 1);
```